bpfd/src/command.rs

```diff
@@ -135,6 +135,14 @@ impl std::fmt::Display for Direction {
     }
 }
 
+#[derive(Debug, Clone)]
+pub(crate) struct ListInfo {
```

Comment:
The more I think on it, the more I don't think this is needed. Just create a ProgramInfo, and nest KernelProgramInfo inside of it.

Reply:
Done, now KernelProgramInfo is an integral part to our whole workflow